modules/cco-ccoctl-upgrading.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc
+
+[id="cco-ccoctl-upgrading_{context}"]
+= Updating AWS resources with the Cloud Credential Operator utility
+
+The process for upgrading an {product-title} cluster configured for manual mode with STS is similar to installing on a cluster for which you create the AWS resources individually.
```

Comment:
Suggestions to spell out STS on the first reference in each module. 
s/STS/AWS Secure Token Service (STS)